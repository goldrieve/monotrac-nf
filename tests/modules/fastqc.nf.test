nextflow_process {

    name "Test Process FASTQC"
    script "main.nf"
    process "FASTQC"

    test("Should run fastqc") {

        when {
            params {
            }
            process {
                """
                input[0] = ["SVP_test", file("${projectDir}/data/syn_reads/SVP.fq.gz")]  
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out.stats).match()
        }

    }

}
