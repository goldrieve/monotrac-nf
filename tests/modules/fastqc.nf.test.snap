{
    "Should run fastqc": {
        "content": [
            [
                "summary.txt:md5,8759ba71da2825a37eae5f55ac890e0c"
            ]
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "25.04.7"
        },
        "timestamp": "2025-10-06T08:43:48.022938324"
    }
}